packages/zowe-explorer/__tests__/__unit__/trees/shared/SharedUtils.unit.test.ts

```diff
@@ -1514,3 +1528,98 @@ describe("SharedUtils.handleProfileChange", () => {
         expect(errorSpy).toHaveBeenCalledWith("error while updating profile on node");
     });
 });
+
+describe("SharedUtils helpers", () => {
+    const mockGetMvsApi = jest.fn();
+    let originalGetMvsApi: any;
+
+    beforeEach(() => {
+        jest.resetAllMocks();
+        jest.clearAllMocks();
+
+        originalGetMvsApi = (ZoweExplorerApiRegister as any).getMvsApi;
+        (ZoweExplorerApiRegister as any).getMvsApi = mockGetMvsApi;
+    });
+
+    afterEach(() => {
+        (ZoweExplorerApiRegister as any).getMvsApi = originalGetMvsApi;
+        jest.restoreAllMocks();
+    });
+
+    test("isSamePhysicalDataset returns true when dsname and vols match", async () => {
```

Comment:
(nitpick) Typically we use `it` statements to write our tests. `test` is synonymous with `it` but it's good to be consistent 😋